tests/lib/app_version/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
`tests/cmake` might be a better location for it (optional)

Reply:
done